4.assess_fibrosis_reversal/scripts/1.vis_probabilities.r

```diff
@@ -0,0 +1,257 @@
+    axis.text.x = element_text(size = 20),
+    axis.title = element_text(size = 24),
+    strip.text = element_text(size = 18),
+    strip.background = element_rect(
+      colour = "black",
+      fill = "#fdfff4"
+    )
+  )
+
+# Save figure
+ggsave(paste0("./figures/", batch_var, "_DMSO_controls_ridge_plot.png"), ridge_plot_healthy, height = height, width = width, dpi = 500)
+
+ridge_plot_healthy
+
+# Custom color palette inspired by Dark2
+custom_palette <- c(
+  "#1b9e77",  # Dark2 color 1
+  "#d95f02",  # Dark2 color 2
+  "#7570b3",  # Dark2 color 3
+  "#e7298a",  # Dark2 color 4
+  "#66a61e",  # Dark2 color 5
+  "#e6ab02",  # Dark2 color 6
+  "#a6761d",  # Dark2 color 7
+  "#666666",  # Dark2 color 8
+  "#b3b3b3",  # Light gray (alternative)
+  "#8dd3c7",  # Teal (alternative)
+  "#fb8072"   # Light coral (alternative)
+)
+
+# Filter out DMSO treatment
+filtered_df <- annotated_combined_probabilities %>% filter(Metadata_treatment != "DMSO")
+
+# Step 1: Calculate the median Healthy_probas for each treatment (you can use mean instead)
+treatment_order <- filtered_df %>%
+  group_by(Metadata_treatment) %>%
+  summarise(median_healthy_proba = median(Healthy_probas)) %>%
+  arrange(desc(median_healthy_proba))  # Sort in descending order of median
+
+# Step 2: Reorder the 'Metadata_treatment' factor levels in reverse order for top-to-bottom plotting
+filtered_df <- filtered_df %>%
+  mutate(Metadata_treatment = factor(Metadata_treatment, levels = rev(treatment_order$Metadata_treatment)))
+
+# Step 3: Create the ridge plot with reordered treatments
+height <- 17
+width <- 18
+options(repr.plot.width = width, repr.plot.height = height)
+
+# Create a ridge plot excluding DMSO and using all other treatments on the y-axis
+ridge_plot_non_DMSO <- ggplot(filtered_df, aes(x = Healthy_probas, y = Metadata_treatment, fill = Pathway)) +
+  geom_density_ridges(alpha = 0.7, scale = 2.25, rel_min_height = 0.01, bandwidth = 0.1) +  # Ridges colored by Pathway
+  geom_vline(xintercept = 1, linetype = "dashed", color = "black") +
+  scale_x_continuous(breaks = seq(0, 1, 0.5)) +
+  labs(x = "Probability of healthy prediction", y = "Treatment") +  # Update axis labels
+  scale_fill_manual(values = custom_palette) +  # Use the custom color palette
+  theme_bw() +
+  theme(
+    legend.position = "right",  # Position the legend on the right
+    legend.title = element_text("Pathway", size = 22),
+    axis.text = element_text(size = 22),
+    axis.text.x = element_text(size = 22),
+    axis.title = element_text(size = 24),
+    strip.text = element_text(size = 18),
+    strip.background = element_rect(
+      colour = "black",
+      fill = "#fdfff4"
+    ),
+    legend.text = element_text(size = 18),  # Adjust legend text size if needed
+    legend.key.size = unit(1.5, "lines")  # Adjust the size of the legend keys
+  ) +
+  facet_wrap(~ Metadata_Plate, ncol = 4)  # Facet by Metadata_Plate, set ncol to a value > 1 for horizontal layout
```

Comment:
Same with these for consistency across figures, consider adding to a themes file

Reply:
Same as above comment, thanks!